apps/search/js/newtab.js

```diff
@@ -1,8 +1,17 @@
 'use strict';
-/* global asyncStorage */
+/* global asyncStorage, SettingsListener */
 
 (function(exports) {
 
+  SettingsListener.observe('browser.private.default', false, function (value) {
+    if (value) {
+      exports.newtab.pretendToBePrivate();
+    }
```

Comment:
nit: prefer the `} else {` style. It's not enforced (but should be), and is that way in most gaia apps.